common/utils.go

```diff
@@ -35,6 +39,14 @@ const (
 	nodekeyFileName  = "nodekey"
 )
 
+func GenerateIPs(num int) (ips []string) {
```

Comment:
How about to defined subnet `10.1.1.xxx` as const value and pass it as a param of `GenerateIPs `?
```
func GenerateIPs(subnet string, num int) (ips []string){
    ....
}
```